src/pages/signin/LoginForm.js

```diff
@@ -126,7 +124,7 @@ class LoginForm extends React.Component {
 
         const login = this.state.login.trim();
         if (!login) {
-            this.setState({formError: 'common.pleaseEnterEmailOrPhoneNumber'});
+            Session.setAccountErrors(0, this.props.translate('common.pleaseEnterEmailOrPhoneNumber'));
```

Comment:
shoot I completely missed these were client side validation errors 🤦 I brought this up here to make sure I didn't just completely mislead you here https://expensify.slack.com/archives/C02HWMSMZEC/p1663178948470959

Reply:
Following the discussion it seem this PR doesn't hold again.